components/suggestion/suggestion_box.jsx

```diff
@@ -318,6 +318,7 @@ export default class SuggestionBox extends React.Component {
         const pretext = textbox.value.substring(0, textbox.selectionStart) + e.data;
 
         this.pretext = pretext;
```

Comment:
This line is no longer needed because is executed inside `handlePretextChanged`

Reply:
Good catch, @jespino! Fixed :)